File: AMS_API_V9/src/main/java/com/cryptosoft/dtos/BatchesOfInstructor.java
package com.cryptosoft.dtos;

import com.cryptosoft.entity.BatchType;
import com.cryptosoft.entity.Department;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class BatchesOfInstructor {

	private Integer batchId;
	private String batchName;
	private BatchType batchType;
	private Department department;
	private String semester;
	
	private Integer CourseId;
	private String CourseName;
}

File: AMS_APP_V1/src/app/models/batches-of-instructor.ts
import { DepartmentResponse } from "./department-response"

export interface BatchesOfInstructor {
     batchId:number;
	 batchName:string;
	 batchType:string;
	 department:DepartmentResponse;
	 semester:string;
	 courseId:number;
	 courseName:string;
}
